static/js/containers/LearnerSearchPage.js

```diff
@@ -61,7 +61,7 @@ class LearnerSearchPage extends React.Component {
       startEmailEdit(
         {
           type: SEARCH_EMAIL_TYPE,
-          params: {query: searchkit.query.query},
+          params: {searchkit: searchkit},
```

Comment:
I'm not sure putting `SearchkitManager` into the redux store is a good idea. Why not just pass `searchkit` when `closeEmailComposerAndSend` is called?